hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -360,17 +423,14 @@ def write_s3_text(txt_content: io.StringIO, bucket, key: str) -> None:
     logging.basicConfig(level=logging.INFO)
     # This will only kinda work for so long - eventually will
     # need to use a proper harness
```

Comment:
can we just delete this? the `if __name__ == "__main__":` code should just be in a test harness if we want to commit it. It super easy to get out of date.